README.md

```diff
@@ -1,2 +1,39 @@
 # Capstone-Project-m1
-Module-1 Capstone Project
+## Description
+- Module-1 Capstone Project.
+## Learning Objectives
+- Use semantic HTML tags.
+- Apply best practices in HTML code.
+- use CSS selectors correctly.
+- Use CSS box model.
+- Use flex box to place elements in the page.
+- Demostrate ability to create UIs adaptable to different screen sizes using media queries.
+- Use GitHub Pages to deploy web pages.
+- Apply JavaScript best practices and language style giudes in code.
+- Use JavaScript to manipulate DOM elements.
+- Use JavaScript events.
+- Use objects to store and access data.
+- communicate technical concepts to other technical people.
+## Built with
+- HTML, CSS, and JAVASCRIPT,
+- Visual studio code, Git and GitHub, GitHub Desktop and Git Bash
+## Prerequisites
+- Setup
+- Install
+- Usage
+- Run tests
+- Deployment.
+## Author(s)
+- GitHub: https://github.com/lugard1
+- Twitter: https://twitter.com/Dsn3kings
+- Linkedin: https://www.linkedin.com/in/lugard-agu-45bb05b6
+## Contributing
+- Contributions, issues, feature requests are welcome!
+## Acknowledgements
+- Hat tip to anyone who supported this work.
+- inspiration.
+- etc
+## License
+- This project is MIT licensed.
+## Update
+- Link to online version of Capstone-Project-m1: https://lugard1.github.io/Capstone-Project-m1
```

Comment:
hello there the following are requirements you missed and will have to address before your project is approved  😄 👍🏾 

### Video presentation

1. you should record a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video should contain a transparent and objective description of the project in professional English. The topic and purpose of the project are made clear.
4. The video contains a project demo, showing the pages in different screen sizes.
5. The video contains a highlight about a piece of the code the student finds attractive or is proud of making.

### Creative Commons Attribution

1. The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).Creative Commons Attribution

### Live version

1. you should make use of GitHub Pages to deploy the web pages.
2. The project is deployed properly, and all pages, links, and images work as expected.
3. A link to the deployed project is present in the README.